Resources/Textures/_RMC14/Objects/Misc/Books/Bibles/personal.rsi/meta.json

```diff
@@ -0,0 +1,22 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from cmss13 at https://github.com/cmss13-devs/cmss13/commit/f165d66c3a8b4ab02e6af02035c28e344d0bac69, edited by SharkSnake98",
```

Comment:
this should link to the DMI file you got the sprites from, not just the commit alone